test/file.jl

```diff
@@ -1156,3 +1156,35 @@ if !Sys.iswindows()
         test_22566()
     end
 end  # !Sys.iswindows
+
+function test_22922()
+    mktempdir() do tmpdir
+        (path, dirname) = splitdir(tmpdir)
```

Comment:
Just use `basename` to get the directory name (but don't call it `dirname` since that's the name of a function returning the equivalent of `path`).